src/library/Utilities/RemoteContainerConnectionDetailsResolver.cs

```diff
@@ -49,6 +49,20 @@ internal class RemoteContainerConnectionDetailsResolver
                 return remoteContainerConnectionDetails;
             }
 
+            // Check if restoration pod is present and in running state. This is an indication that previous session is still connected or it has not finished restoring yet.
+            var allPods = (await _kubernetesClient.ListPodsInNamespaceAsync(remoteContainerConnectionDetails.NamespaceName, null, cancellationToken))?.Items;

```

Comment:
Hi, just noticed this, instead of var if we know the type of variable we can instead declare it with the exact type which is safer than var keyword. 
stack over flow reference - https://stackoverflow.com/questions/545616/why-would-var-be-a-bad-thing
```suggestion
 V1PodList v1 = await _kubernetesClient.ListPodsInNamespaceAsync(namespaceName, selectors, cancellationToken);
                IList<V1Pod> items = v1.Items;
                if (items.Any() || items.Count > 0)
                {
// your code here
                }
            var allPods = (await _kubernetesClient.ListPodsInNamespaceAsync(remoteContainerConnectionDetails.NamespaceName, null, cancellationToken))?.Items;

```

Reply:
@elenavillamil i think you missed my comment. No worries it's just a suggestion Which would increase the quality of code. :)